### PR TITLE
Add original_reason_entitlement_code to Eligibility

### LINKED ISSUE #26: Add an underscore to LDS connector schema
Add an underscore to the schema(s) that are created by the LDS connector.  This will help organize the data seperately from The Tuva Project.

e.g. Medical_claim should materializing in a schema called `_tuva_claims` instead of `tuva_claims`

### PR DESCRIPTION
**Pull request by sarah-tuva**
## Describe your changes
* Added new field original_reason_entitlement_code
* Added under underscore prefix to tables created by the connector
* Added claims_enabled var
* Updated project version
* Updated dbt docs

Also resolves #26 

## How has this been tested?
Installed The Tuva Project package from [branch](https://github.com/tuva-health/the_tuva_project/tree/update-eligibility).
Ran dbt build and validated output.

## Reviewer focus
Please summarize the specific items you’d like the reviewer(s) to look into.

## Checklist before requesting a review
- [ ]  (Optional) I have recorded a Loom performing a self-review of my code
- [ ]  My code follows style guidelines
- [ ]  I have commented my code as necessary
- [ ]  (New models only) I have implemented generic dbt tests to validate primary keys/uniqueness in my model
- [x]  I have added at least one Github label to this PR

### CODE REVIEW

**Comment by sarah-tuva** [comment]
Loom update on the test failure in Redshift. https://www.loom.com/share/472e24823cb34eb589a1d775746b0de8

**Comment by sarah-tuva** [comment]
Relevant bug cards created for the bugs discussed in the loom.

https://github.com/tuva-health/the_tuva_project/issues/237
https://github.com/tuva-health/the_tuva_project/issues/238

File: models/_models.yml
Comment (thutuva) [inline models/_models.yml:1]: Nice catch on adding an underscore if someone uses the tuva_schema_prefix var

**Review by thutuva — APPROVED** [review]
Looks great.

File: models/_models.yml
Comment (sarah-tuva) [reply models/_models.yml:1]: @thutuva Thanks for reviewing! 🙏

**Pull request merged**